lib/evaluation/handlers/subjectIsIssuerEvaluationHandler.ts

```diff
@@ -47,7 +47,7 @@ export class SubjectIsIssuerEvaluationHandler extends AbstractEvaluationHandler
           [currentDescriptor.path]
         ) as { path: PathComponent[]; value: ICredential }[];
         //TODO: ESSIFI-186
-        if (vc[0]?.value.issuer === (vc[0]?.value.credentialSubject as ICredentialSubject).id) {
+        if (getSubjectIdsAsString(vc[0]?.value).indexOf(getIssuerString(vc[0]?.value)) !== -1) {
```

Comment:
This could result in getSubjectIdAsString being called with undefined, which would go wrong as that accesses credentialSubject on what is undefined